src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -154,6 +154,9 @@ public void replacePlayer(@NotNull TabPlayer tabPlayer) {
 
     // Update a player's name in the tab list
     public void updatePlayer(@NotNull TabPlayer tabPlayer) {
+        if (!tabPlayer.getPlayer().isActive()){
```

Comment:
style: could you just add a space between the last close parenthesis and the open curly parenthesis here?